testData/testMeGenerator/mockReturned/src/com/example/services/impl/Foo.java

```diff
@@ -12,10 +12,8 @@ public class Foo{
 
     private FooFighter fooFighter;
     private Supplier<Integer> result;
-    private Logger logger;
```

Comment:
I think we should keep the tested loggers, even if it doesn't seem as a good feature to generate verification statements on log calls. that's the current behavior and we better not hide it. anyway in this specific use case logger must be at least mocked to avoid NPE. 
going forward, would be good to support a feature for avoiding mock verification statements for loggers.

Reply:
i add the logger back, i am trying to add a feature to support DI and constructors, in that case the logger won't be added